src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/IExcludeDate.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.model.entity
+
+import java.time.LocalDate
+
+interface IExcludeDate {
```

Comment:
Nice idea, but do we need this? It seems to only be 2 duplicate fields, and likely won't be implemented by any other entities?

Happy to leave decision to you 👍

Reply:
Will keep it. Its just to make things generic in ExcludeDateService.